doc/topics/releases/3006.rst

```diff
@@ -0,0 +1,137 @@
+| Raspbian 11  |         no          |        yes          |
++--------------+---------------------+---------------------+
+| Fedora 35    |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| Fedora 36    |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| MacOS        |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| Windows      |         yes         |        yes          |
++--------------+---------------------+---------------------+
+
+
+Repo paths
+----------
+
++----------+-----------------------------------------------+-----------------------------------------+
+|     OS   | Tiamat Path                                   | Non-Tiamat Path                         |
++==========+===============================================+=========================================+
+| RHEL     | https://repo.saltproject.io/salt/py3/redhat/  | https://repo.saltproject.io/py3/redhat/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Ubuntu   | https://repo.saltproject.io/salt/py3/ubuntu/  | https://repo.saltproject.io/py3/ubuntu/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Debian   | https://repo.saltproject.io/salt/py3/debian/  | https://repo.saltproject.io/py3/debian/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Raspbian | https://repo.saltproject.io/salt/py3/debian/  | https://repo.saltproject.io/py3/debian/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Fedora   | Hosted on Fedora Repos                        | Hosted on Fedora Repos                  |
++----------+-----------------------------------------------+-----------------------------------------+
+| MacOS    | https://repo.saltproject.io/salt/py3/osx/     | https://repo.saltproject.io/osx/        |
++----------+-----------------------------------------------+-----------------------------------------+
+| Windows  | https://repo.saltproject.io/salt/py3/windows/ |  https://repo.saltproject.io/windows/   |
++----------+-----------------------------------------------+-----------------------------------------+
+
+Note that the Tiamat paths above will not be available until the day of the Phosphorus release.
+
+
+My OS only provides Tiamat packages. What do I do?
+--------------------------------------------------
+If you are using a platform that no longer provides the non-Tiamat packages as shown
+in the previous tables, you will need to update your repo files to point to the new Tiamat packages.
+The new repo paths are provided in the previous tables, but the full instructions will be included
+at https://repo.saltproject.io.
```

Comment:
**Suggestion:** Same suggestion as above. The repo install instructions will be deprecated over time, so we'll want to link to the new guide.